docs/extensions/graph/users.md

```diff
@@ -163,3 +163,27 @@ to be updated. E.g. to update the `displayName` Attribute:
 ```
 
 Similar to creating a user via `POST`,  the `PATCH` request will return the user object containing the new attribute values.
+
+### Change password
+
+#### `POST /me/changePassword`
+
+Users can change their own password by sending a POST request to `/me/changePassword`
```

Comment:
Question: Why `POST`? Wouldn't that be a `PATCH`?

Reply:
We try to follow what MS is doing. They're using `POST` https://github.com/owncloud/ocis/issues/3063

Reply:
Ah, sure thats graph, sorry. Then we go with `POST` of course. (Allthough it's wrong :smile: )